docs/source/concepts/bento.rst

```diff
@@ -965,6 +965,38 @@ Setup script is always executed after the specified Python packages, conda depen
 and system packages are installed. Thus user can import and utilize those libraries in
 their setup script for the initialization process.
 
+Enable features for your Bento
+""""""""""""""""""""""""""""""
+
+Users can optionally pass in ``--enable-features`` flag to ``bentoml containerize`` to
+enable additional features for the generated Bento container image.
+
+This maps directly 1-to-1 with BentoML's pip installable features. For example:
```

Comment:
I don't think this is helpful to state unless you're linking to external docs listing the additional features. Let's just list the features and maybe a quick description of what it's for?

Reply:
sg